svcb_test.go

```diff
@@ -70,10 +72,12 @@ func TestDecodeBadSVCB(t *testing.T) {
 			key:  SVCB_ALPN,
 			data: []byte{3, 0, 0}, // There aren't three octets after 3
 		},
-		{
+		// The caller is responsible for ensuring the buffer is empty after
+		// unpacking, see unpackDataSVCB.
+		/*{
```

Comment:
why this change?

Reply:
That's because I moved the error condition out of `(*SVCB).unpack`, I've moved it back so I can restore this test case again.